scalafix-core/src/main/scala/scalafix/internal/config/ScalafixMetaconfigReaders.scala

```diff
@@ -41,11 +41,8 @@ trait ScalafixMetaconfigReaders {
       Default,
       Scala211,
       Scala212,
-      Sbt0137,
-      Sbt1,
-      Dotty,
-      Paradise211,
-      Paradise212
+      Scala213,
+      Scala3
```

Comment:
what happened to `Sbt1` ? I guess it could be useful if someone has a sbt-type file with a different extension than `.sbt` (very far-fetched)

Reply:
even the old code don't really take `sbt` dialect into account.
sbt dialect is always 213 (which is the default) with allowTopLevelItems. 
and now sbtDialct is `val DefaultSbtDialect: Dialect = scala.meta.dialects.Sbt1`

  private val sbtDialect = dialect.copy(allowToplevelTerms = true)

  def dialectForFile(path: String): Dialect =
    if (path.endsWith(".sbt")) sbtDialect
    else dialect

So users are supposed to configure either scala213 or scala3 in dialect. Do you think we should restrict to thode two values? 
  implicit lazy val dialectReader: ConfDecoder[Dialect] = {
    import ScalafixConfig.{DefaultDialect => Default}
    import scala.meta.dialects._
    ReaderUtil.oneOf[Dialect](
      Scala213,
      Scala3,
      //Scala211
      //Scala212
    )
  }

Reply:
> Do you think we should restrict to those two values?

If we do, then it would be `Scala2` and `Scala3`, no?

Reply:
> By the way, I thought sbt1 is just sbt with a higher version 1.x.x
> Isn't that the case?

It is, the format and binary backward compatibility is frozen in 1.x

Reply:
> even the old code don't really take sbt dialect into account.
> sbt dialect is always 213 (which is the default) with allowTopLevelItems.

Interesting! I think we should rather use the dialect defined in scalameta than defning our own variant here, no?
https://github.com/scalameta/scalameta/blob/6c0a370cc278d473588f973c436d8378a691d2b1/scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala#L70-L72

Reply:
yes! It's what I did. I Like the idea of having only scala2 and Scala3 dialect !!